### PR TITLE
clean up error codes explanation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0220.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-You used an associated type which isn't defined in the trait.
+A used associated type wasn't defined in the trait.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0220.md]:
```suggestion
An associated type was used which was not defined in the trait.
```

Comment (GuillaumeGomez) [reply src/librustc_error_codes/error_codes/E0220.md]: This sentence makes no sentence. :p

Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0220.md]:
```suggestion
The associated type used was not defined in the trait.
```

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

**Comment by GuillaumeGomez** [comment]
Updated!

**Pull request merged**